_data/en/contribute.yml

```diff
@@ -1,22 +1,22 @@
 name: Contribute
 items:
-  - title: Become an active member of javaBin?
+  - title: Want to become an active member of javaBin?
     content:
-        Want to join actively in javaBin and organize meetups and JavaZone? In Oslo, we meet every Tuesday
-        18:00 PM at <a href="https://www.google.com/maps/place/Pilestredet+56,+0167+Oslo">Teknologihuset</a>,
-        just show up at one of our evenings. In the regions we meet less frequently - show up at a
-        meetup event and talk with us.
+        Want to join javaBin and help us organizing meetups and JavaZone? In Oslo, we meet every Tuesday
+        18:00 PM at <a href="https://www.google.com/maps/place/Pilestredet+56,+0167+Oslo">Teknologihuset</a>.
```

Comment:
Ok, how about "In Oslo we meet at 18:00 every Tuesday at [Teknologihuset](https://www.google.com/maps/place/Pilestredet+56,+0167+Oslo); other regions meet less frequently, so [check out your regions upcoming events](http://java.no/#locations) for more details. Just show up at any of the meetings or scheduled meetups and say hi!"